src/main/java/io/r2dbc/postgresql/codec/UuidArrayCodec.java

```diff
@@ -0,0 +1,44 @@
+package io.r2dbc.postgresql.codec;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.r2dbc.postgresql.message.Format;
+import io.r2dbc.postgresql.type.PostgresqlObjectId;
+import io.r2dbc.postgresql.client.Parameter;
+import io.r2dbc.postgresql.util.Assert;
+import reactor.util.annotation.Nullable;
+
+import java.util.UUID;
+import java.util.function.Supplier;
+
+public class UuidArrayCodec extends AbstractArrayCodec<UUID> {
```

Comment:
Nit: codecs are typically package-private and `final`.

Reply:
Ok

Reply:
Flipped to local final